### PR TITLE
standardize on using percentages not decimals

### PR DESCRIPTION
**Pull request by hubcity**
Once you merge the state taxes pull request I will probably need to revisit this one.

The switch to tomllib is in case the users put in something like
[[0, 10.0]]

With python <3.11 and toml this throws a heterogeneous type error, but with tomllib it works.
I based this change on https://github.com/jendrikseipp/vulture/issues/339

### CODE REVIEW

**Comment by hubcity** [comment]
Something about this is broken, but I don't know what.

**Comment by wscott** [comment]
I assume that using 0.0 would fix it?

File: src/fplan/fplan.py [hunk 1]
```diff
@@ -43,15 +43,18 @@ def load_file(self, file):
...
+        tmp_taxrates = d.get('taxrates',
+                              [[0,      10],
+                               [22000,  12],
+                               [89450 , 22],
+                               [190750, 24],
+                               [364200, 32],
+                               [462500, 35],
+                               [693750, 37]])
+        # add fake level and swith to decimals
+        tmp_taxrates[:0] = [[0, 0]]
```
Comment (hubcity) [inline src/fplan/fplan.py:55]: This is the line that adds the [0,0] level so we don't have to ask the users to put in a fake level.

**Comment by hubcity** [comment]
I moved the decimal vs whole number fix to a different PR.  Everything in the PR is whole numbers and calculating correctly, I think.

**Comment by wscott** [comment]
I did a quick merge, but clearly, these config values need to be under the same `[taxes]` section as well now.
Also, how about getting rid of the `sample_taxrates` file and include the 'single' table as a comment in the example file.

File: src/fplan/fplan.py [hunk 2]
```diff
@@ -193,7 +197,8 @@ def solve(args):
 
         (taxbase, last_cut, last_rate) = (0, 0, 0)
         for (cut, rate) in S.taxrates:
-            rate += S.state_tax
+            if rate > 0:                             # if below fed std_ded, assumes tax 0%
```
Comment (wscott) [inline src/fplan/fplan.py:200]: This tweak needs to be copied to the print function below.

Comment (hubcity) [reply src/fplan/fplan.py:200]: Assuming that the federal taxes don't add a 0% bracket at a non-zero dollar amount, the practical effect of this change is to add a [0, 0] bracket to state taxes, just like we do with federal taxes.  It ensures that we don't get a state refund for staying under the standard deduction.

The print function already makes an equivalent assumption in lines 408-409 where it sets the inc (and therefore taxes) to 0 if it is below the standard deduction.

The problem with this code is that it assumes that the state standard deduction is the same as the federal standard deduction.  That is not universally true.  Maybe we need another configuration parameter (I think there was a state deduction variable, but it wasn't used anywhere).  We could try to make the code clearer if we add that.

Comment (hubcity) [reply src/fplan/fplan.py:200]: Ok, ignore most of my last comment.  Your fix was needed.

**Pull request merged**